src/test/java/core/basesyntax/service/strategy/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax.service.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.MapStorage;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.FruitTransactionProcessorImpl;
+import core.basesyntax.service.strategy.OperationHandlerStrategy;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationHandlerTest {
+    private static final String TEST_FRUIT_NAME = "watermelon";
+    private static final int TEST_QUANTITY = 50;
+    private static FruitTransactionProcessorImpl fruitTransactionProcessor;
+    private static Storage storage;
+
+    @BeforeEach

```

Comment:
The same mistake as was in previous version of tests implementation
```suggestion
    @BeforeAll
```
please fix in all test classes.